tests_new/integration_tests/modules/s3_datasets/global_conftest.py

```diff
@@ -288,20 +400,72 @@ def persistent_s3_dataset1(client1, group1, persistent_env1, testdata):
 
 
 @pytest.fixture(scope='session')
-def persistent_imported_sse_s3_dataset1(client1, group1, persistent_env1, testdata):
+def persistent_imported_sse_s3_dataset1(client1, group1, persistent_env1, persistent_env1_aws_client, testdata):
+    bucket_name = 'dataalltestingpersistentimportedsses3'
```

Comment:
won't this cause issues with globally unqiue requirement of S3 Bucket names?

Reply:
Yes, I ran into that when testing the #1391. It should be fixed now